tests/test_meshtastic_utils_coverage.py

```
@@ -1683,15 +1683,28 @@ def test_on_lost_meshtastic_connection_clears_ble_future_globals(self):
         mu.event_loop.is_closed.return_value = False
         mu.reconnecting = False
 
-        with patch("mmrelay.meshtastic_utils.reconnect"):
-            with patch("mmrelay.meshtastic_utils.logger"):
-                mu.on_lost_meshtastic_connection(detection_source="test source")
+        def _submit_reconnect_coro(coro, _loop):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_submit_reconnect_coro`

(ANN202)